less/pages/open-web-fellows/shared.less

```diff
@@ -0,0 +1,41 @@
+.home-page, .fellows-page, .info-page,  .overview-page {
+  .content-container {
+    .content {
+      padding: 85px 0 70px 0;
+    }
+    @media screen and (max-width: 767px) {
+      .content {
+        padding: 25px;
+      }
+    }
+  }
+  h1 {
+    line-height: 77px;
+    font-size: 64px;
+    color: #80B6B1;
+    margin: 16px;
+    @media screen and (max-width: 767px) {
+      margin-top: 48px;
+      font-size: 34px;
+      line-height: 40px;
+    }
+  }
+
+  h2 {
+    font-size: 50px;
+    font-weight: 700;
+    @media screen and (max-width: 767px) {
+      font-size: 32px;
+      font-weight: 400;
+    }
+  }
+
+  h3 {
+    font-size: 32px;
+  }
+
+  h4 {
+    font-size: 20px;
+    font-weight: 300;
+  }
+}
```

Comment:
All these changes were specific to the webfellows page, but classes in my new page had some of these styles bleeding into it. So I had to and should pull them out.